Remove the --dynamic-linker option (which doesn't exist) from build options.

From: Roman Donchenko <roman.donchenko@itseez.com>


---
 .../Samples/SamplesAssistant/jni/Android.mk        |    2 +-
 Platform/Android/jni/OpenNI.jni/Android.mk         |    2 +-
 Platform/Android/jni/OpenNI/Android.mk             |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Platform/Android/Samples/SamplesAssistant/jni/Android.mk b/Platform/Android/Samples/SamplesAssistant/jni/Android.mk
index e4b980d..9515e8d 100644
--- a/Platform/Android/Samples/SamplesAssistant/jni/Android.mk
+++ b/Platform/Android/Samples/SamplesAssistant/jni/Android.mk
@@ -25,7 +25,7 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_CFLAGS:= -fvisibility=hidden -DXN_EXPORTS
 
-LOCAL_LDFLAGS += -Wl,--export-dynamic --dynamic-linker
+LOCAL_LDFLAGS += -Wl,--export-dynamic
 
 LOCAL_LDLIBS := -llog
 LOCAL_LDLIBS += $(OPENNI_DIR)/Platform/Android/libs/armeabi-v7a/libOpenNI.so
diff --git a/Platform/Android/jni/OpenNI.jni/Android.mk b/Platform/Android/jni/OpenNI.jni/Android.mk
index 30b332c..30ce781 100644
--- a/Platform/Android/jni/OpenNI.jni/Android.mk
+++ b/Platform/Android/jni/OpenNI.jni/Android.mk
@@ -22,7 +22,7 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_CFLAGS:= -fvisibility=hidden -DXN_EXPORTS
 
-LOCAL_LDFLAGS += -Wl,--export-dynamic --dynamic-linker
+LOCAL_LDFLAGS += -Wl,--export-dynamic
 
 LOCAL_LDLIBS := -llog
 
diff --git a/Platform/Android/jni/OpenNI/Android.mk b/Platform/Android/jni/OpenNI/Android.mk
index 47b0911..355cb74 100644
--- a/Platform/Android/jni/OpenNI/Android.mk
+++ b/Platform/Android/jni/OpenNI/Android.mk
@@ -27,7 +27,7 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_CFLAGS:= -fvisibility=hidden -DXN_EXPORTS
 
-LOCAL_LDFLAGS += -Wl,--export-dynamic --dynamic-linker 
+LOCAL_LDFLAGS += -Wl,--export-dynamic
 
 LOCAL_EXPORT_C_INCLUDES := $(LOCAL_PATH)/../../../../Include
 LOCAL_SHARED_LIBRARIES := libusb-1.0
